sdk/objectanchors/Azure.MixedReality.ObjectAnchors/src/AssetConversionOperation.cs

```diff
@@ -0,0 +1,148 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Azure.Core;
+using Azure.Core.Pipeline;
+
+namespace Azure.MixedReality.ObjectAnchors
+{
+    /// <summary> A long running asset conversion process for Object Anchors</summary>
+    public class AssetConversionOperation : Operation<AssetConversionProperties>
+    {
+        private static readonly TimeSpan defaultPollingInterval = TimeSpan.FromSeconds(15);
+        private readonly ClientDiagnostics _diagnostics;
+        private readonly ObjectAnchorsClient _objectAnchorsClient;
+        private readonly Guid _jobId;
+        private Response _lastConversionResponse;
+        private Response<AssetConversionProperties> _conclusiveConversionResponse;
+        private bool _conversionEnded;
+        private object _updateStatusLock = new object();
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="AssetConversionOperation"/> class.
+        /// You must call <see cref="UpdateStatus(CancellationToken)"/> or <see cref="UpdateStatusAsync(CancellationToken)"/> before you can get the <see cref="Value"/>.
+        /// </summary>
+        /// <param name="jobId">The ID of this operation.</param>
+        /// <param name="objectAnchorsClient">The client used to check for completion.</param>
+        public AssetConversionOperation(Guid jobId, ObjectAnchorsClient objectAnchorsClient) : this(jobId, objectAnchorsClient, objectAnchorsClient._clientDiagnostics)
```

Comment:
```suggestion
        public AssetConversionOperation(Guid jobId, ObjectAnchorsClient objectAnchorsClient)
            : this(jobId, objectAnchorsClient, objectAnchorsClient._clientDiagnostics)
```

Reply:
Pending

Reply:
Constructor no longer present, solution not applicable anymore